RELEASE/scripts/autoscend/combat/auto_combat_util.ash

```diff
@@ -613,6 +613,13 @@ string yellowRayCombatString(monster target, boolean inCombat, boolean noForceDr
 			return auto_combatSaberYR();
 		}
 	}
+	
+	// shocking lick doesn't cause everything looks yellow effect and limited only by how many batteries you have. Use all other sources first.
+	boolean simulateBatteryCombine = true; //if we don't have charges, see if a 9-volt can be made
+	if(inCombat ? have_skill($skill[Shocking Lick]) : (get_property("shockingLickCharges").to_int() > 0 || batteryCombine($item[battery (9-Volt)], simulateBatteryCombine)))
```

Comment:
```suggestion
	if(inCombat ? have_skill($skill[Shocking Lick]) : (get_property("shockingLickCharges").to_int() > 0 || can_get_battery($item[battery (9-Volt))))
```
batteryCombine simulate mode should only be used in the function can_get_battery.
can_get_battery should be used to see if we can get said battery.

the reason for the distinction is that battery combine is only used for combining and not untinkering. 
auto_getBattery is how you acquire a battery. and that can either combine or untinker or both.

I am not 100% if I should leave it as can_get_battery or if I should add a simulate option to auto_getBattery. which do you think would be clearer?